hermes-common/src/main/java/pl/allegro/tech/hermes/common/broker/KafkaOffsetsStorage.java

```diff
@@ -0,0 +1,93 @@
+package pl.allegro.tech.hermes.common.broker;
+
+import jersey.repackaged.com.google.common.collect.Lists;
+import kafka.common.OffsetAndMetadata;
+import kafka.common.OffsetMetadataAndError;
+import kafka.common.TopicAndPartition;
+import kafka.javaapi.OffsetCommitRequest;
+import kafka.javaapi.OffsetCommitResponse;
+import kafka.javaapi.OffsetFetchRequest;
+import kafka.javaapi.OffsetFetchResponse;
+import kafka.network.BlockingChannel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.allegro.tech.hermes.api.Subscription;
+import pl.allegro.tech.hermes.api.TopicName;
+import pl.allegro.tech.hermes.common.time.Clock;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class KafkaOffsetsStorage implements OffsetsStorage {
```

Comment:
This class and ZkOffsetStorage are used only in consumers module. Why they are declared in common module?